doc/sphinx/Pacemaker_Explained/alerts.rst

```diff
@@ -275,7 +275,53 @@ for the full set of instance attributes they support.
             </alert>
          </alerts>
       </configuration>
-   
+
+.. note:: **SNMP alert agent attributes**
+
+   The ``timestamp-format`` meta-attribute should always be set to
+   ``%Y-%m-%d,%H:%M:%S.%01N`` when using the SNMP agent, to match the SNMP
+   standard.
+
+   The SNMP agent provides a number of instance attributes in addition to the
+   one used in the example above. The most useful are ``trap_version``, which
+   defaults to ``2c``, and ``trap_community``, which defaults to ``public``.
+   See the source code for more details.
+
+.. topic:: Sending cluster events as SNMP v3 traps
+
+   .. code-block:: xml
+
+      <configuration>
+         <alerts>
+            <alert id="snmp_alert" path="/path/to/alert_snmp.sh">
+               <instance_attributes id="config_for_alert_snmp">
+                  <nvpair id="trap_node_states" name="trap_node_states"
+                          value="all"/>
+                  <nvpair id="trap_version" name="trap_version" value="3"/>
+                  <nvpair id="trap_community" name="trap_community" value=""/>
+                  <nvpair id="trap_options" name="trap_options"
+                          value="-l authNoPriv -a MD5 -u testuser -A secret1"/>
+               </instance_attributes>
+               <meta_attributes id="config_for_timestamp">
+                  <nvpair id="ts_fmt" name="timestamp-format"
+                          value="%Y-%m-%d,%H:%M:%S.%01N"/>
+               </meta_attributes>
+               <recipient id="snmp_destination" value="192.168.1.2"/>
+            </alert>
+         </alerts>
+      </configuration>
+
+.. note:: **SNMP v3 trap options**
+
+   To use SNMP v3, ``trap_version`` must be set to ``3``, and
+   ``trap_community`` must be set to the empty string.
+
+   The example above uses the ``trap_options`` instance attribute to override
+   the security level, authentication protocol, authentication user, and
+   authentication password from snmp.conf on the command line. Passing the
+   password on the command line is considered insecure. Pass authentication
+   and privacy options suitable for your deployment.
+
```

Comment:
The way these two paragraphs are indented, the second one will also be included in the note.  I think it'd make more sense to either move the second paragraph up above the note or add a second note with a different title for it.

Reply:
That's intentional, both paragraphs are about different trap options

Reply:
Updated commit